.gitignore

```diff
@@ -38,6 +38,20 @@ crashlytics.properties
 crashlytics-build.properties
 fabric.properties
 
+# Database entities
+backend/com.tngtech.apicenter.backend.connector.database.entity.SpecificationEntity
```

Comment:
None of these changes are needed after the other PR is merged, right?

Reply:
Correct that this change is also in the other PR. However that did not make these addtions to the gitignore here not useful.